streamwebs_frontend/streamwebs/models.py

```diff
@@ -680,3 +680,69 @@ def __str__(self):
     class Meta:
         verbose_name = 'canopy cover survey'
         verbose_name_plural = 'canopy cover surveys'
+
+
+@python_2_unicode_compatible
+class Soil_Survey(models.Model):
+    school = models.CharField(max_length=250)
+    teacher = models.CharField(max_length=250)
+    date = models.DateTimeField(default=timezone.now)
+    weather = models.CharField(max_length=250)
+    site = models.ForeignKey(Site, null=True, on_delete=models.CASCADE)
+
+    landscape_pos_choices = [
+        ('summit', 'Summit'),
+        ('slope', 'Slope'),
+        ('depression', 'Depression'),
+        ('large_flat', 'Large Flat Area'),
+        ('stream_bank', 'Stream Bank')
+    ]
+
+    cover_type_choices = [
+        ('bare_soil', 'Bare Soil'),
+        ('rocks', 'Rocks'),
+        ('grass', 'Grass'),
+        ('shrubs', 'Shrubs'),
+        ('trees', 'Trees')
+    ]
+
+    land_use_choices = [
+        ('urban', 'Urban'),
+        ('agricultural', 'Agricultural'),
+        ('recreation', 'Recreation'),
+        ('wilderness', 'Wilderness'),
+        ('other', 'Other')
+    ]
+
+    landscape_pos = models.CharField(max_length=1, default=None,
```

Comment:
@leian7 Good catch! I remember looking it up earlier and not finding any info regarding max_length and fields with choices, so I wasn't sure it mattered. It makes sense now, thanks.